resources/language/resource.language.nl_nl/strings.po

```diff
@@ -596,65 +596,65 @@ msgstr "Sorteer resultaten op"
 
 msgctxt "#30150"
 msgid "Alphabetical order (A-Z)"
-msgstr "Alfabetische lijst (A-Z)"
+msgstr "Alfabetische volgorde (A-Z)"
 
 msgctxt "#30151"
 msgid "Alphabetical order (Z-A)"
-msgstr "Alfabetische lijst (Z-A)"
+msgstr "Alfabetische volgorde (Z-A)"
 
 msgctxt "#30152"
 msgid "Suggestions for you"
-msgstr "Suggesties voor jou"
+msgstr "Aanbevolen voor jou"
 
 msgctxt "#30153"
 msgid "Year released"
-msgstr "Jaar van vrijgave"
+msgstr "Jaar vrijgegeven"
```

Comment:
I would use "Jaar van uitgave", I think this is a better wording.

Reply:
Im sorry im kinda new to this.

Reply:
I searched for examples where movies are "uitgegeven". Books are, movies are not. So i dont agree.

Reply:
You might be right, but vrijgeven sounds odd for a movie, this is more used in context of something we get that has been released to the public. Maybe uitgebracht is better, but "Jaar van uitbrenging" sounds just silly :)